lib/src/platform_radio.dart

```diff
@@ -0,0 +1,163 @@
+/*
+ * flutter_platform_widgets
+ * Copyright (c) 2018 Lance Johnstone. All rights reserved.
+ * See LICENSE for distribution and usage details.
+ */
+
+import 'package:flutter/cupertino.dart' show CupertinoRadio;
+import 'package:flutter/material.dart'
+    show Radio, MaterialTapTargetSize, MaterialStateProperty, VisualDensity;
+import 'package:flutter/widgets.dart';
+
+import 'platform.dart';
+import 'widget_base.dart';
+
+abstract class _BaseData<T> {
+  _BaseData({
+    this.widgetKey,
+    this.value,
+    this.groupValue,
+    this.onChanged,
+    this.toggleable = false,
```

Comment:
```suggestion
    this.toggleable,
```
It cannot be preset since it will interfere with the widget property `data.toggleable ?? toggleable`